AdminWebsite/AdminWebsite.UnitTests/Helper/AdministratorRoleClaimsHelperTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using AdminWebsite.Helper;
+using AdminWebsite.Services.Models;
+using FluentAssertions;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+
+namespace AdminWebsite.UnitTests.Helper
+{
+    public class AdministratorRoleClaimsHelperTest
+    {
+        [Test]
+        public void GetAdministratorClaims_Returns_Claims()
+        {
+            var userGroupData = new UserGroupData
+            {
+                UserRole = "Role1",
+                CaseTypes = new List<string> { "Case1", "Case2" }
+            };
+
+            var claims = new AdministratorRoleClaimsHelper(userGroupData).GetAdministratorClaims().ToList();
+
+            claims.Should().NotBeNull();
```

Comment:
More a preference thing but, to reduce the code, I generally find that if you're testing that it contains an element further down or using the `claims` variable, you'll in a sense get the assertion "for free". I mean, the test will fail with a stacktrace of exactly what was null if it isn't set. Same with the count `HaveCountGreaterThan(0)`, that'll be implicitly checked when you're pulling out a `FirstOrDefault`

Reply:
True but I normally like to know on what assertion failed quickly as opposed to reading the stack trace from an exception that have been thrown... It's sort of like Fail Fast ....maybe..